Mathlib/Algebra/Hom/Equiv/Basic.lean

```diff
@@ -515,15 +515,31 @@ theorem map_ne_one_iff {M N} [MulOneClass M] [MulOneClass N] (h : M ≃* N) {x :
 #align mul_equiv.map_ne_one_iff MulEquiv.map_ne_one_iff
 #align add_equiv.map_ne_zero_iff AddEquiv.map_ne_zero_iff
 
+-- porting note: Mathlib 3 had `@[simps apply]` but right now in Lean 4 it's generating
+-- simp lemmas which don't lint.
 /-- A bijective `Semigroup` homomorphism is an isomorphism -/
-@[to_additive "A bijective `AddSemigroup` homomorphism is an isomorphism", simps apply]
+@[to_additive "A bijective `AddSemigroup` homomorphism is an isomorphism"]
 noncomputable def ofBijective {M N F} [Mul M] [Mul N] [MulHomClass F M N]
 (f : F) (hf : Function.Bijective f) :
     M ≃* N :=
   { Equiv.ofBijective f hf with map_mul' := map_mul f }
 #align mul_equiv.of_bijective MulEquiv.ofBijective
 #align add_equiv.of_bijective AddEquiv.ofBijective
 
+-- porting note: `@[simps apply]` should be making this lemma but it actually makes
+-- a lemma with `toFun` which isn't in simp normal form.
+@[simp] theorem ofBijective_apply {M N F} [Mul M] [Mul N] [MulHomClass F M N]
+    (f : F) (hf : Function.Bijective f) (a : M) : (MulEquiv.ofBijective f hf).toEquiv a = f a :=
+  rfl
+#align mul_equiv.of_bijective_apply MulEquiv.ofBijective_apply
+
+-- porting note: `@[to_additive, simps apply]` should be making this lemma but it
+-- doesn't get generated.
+@[simp] theorem _root_.AddEquiv.ofBijective_apply {M N F} [Add M] [Add N] [AddHomClass F M N]
+    (f : F) (hf : Function.Bijective f) (a : M) : (AddEquiv.ofBijective f hf).toEquiv a = f a :=
+  rfl
+#align add_equiv.of_bijective_apply AddEquiv.ofBijective_apply
```

Comment:
Can `to_additive` generate the additive lemma? In any case, you should use the tag so that proofs that use `MulEquiv.ofBijective_apply` work with `to_additive`.

Reply:
Yes! Thanks!